gtsfm/frontend/detector/detector_base.py

```diff
@@ -10,16 +10,17 @@
 from gtsfm.common.image import Image
 from gtsfm.common.keypoints import Keypoints
 
+DEFAULT_MAX_KEYPOINTS = 8192
```

Comment:
looks good. maybe you can add a URL with line number referencing where COLMAP sets this?